src/pokemon_storage_system.c

```diff
@@ -263,6 +263,8 @@ enum {
 // between 2 Pokémon with held items
 #define MAX_ITEM_ICONS 3
 
+#define MAX_MENU_ITEMS 7
```

Comment:
Constants for array sizes that are only used in the array definition are unnecessary.